C_CPP/Data Structures/Binary Tree/Binary_Tree_Basics

```diff
@@ -0,0 +1,180 @@
+# Tress
+
+As we all know tree is non-linear data structure which is used to represent data containing a hierarchical relationalship between elements.
+
+There are basically five types of trees:-
+
+1. General Tree
+2. Binary Tree
+3. Binary Search Tree 
+4. AVL Tree
+5. B Tree
+
+## Binary Trees
+
+A binary tree *T* is defined as a finite set of elements, called nodes such that:
+
+>T is an empty or null tree
```

Comment:
> are not allowed in md

Reply:
Sir I made few changes in file now. And sorry for the late reply.